components/uuf-renderablecreator-hbs/src/main/java/org/wso2/carbon/uuf/renderablecreator/hbs/impl/js/LoggerObject.java

```diff
@@ -74,6 +76,23 @@ public void error(Object obj) {
         logger.error(getLogMessage(obj));
     }
 
+    public void error(String message, Object obj) {
```

Comment:
If it's wrong why don't you throw an error? We can't teach all the people the best practice. and the variable name is obj? that's not nice.